### PR TITLE
Update sprite with topic 3D

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by oterral** [comment]
No css with it?

**Comment by pfanguin** [comment]
No CSS change, only the png

**Comment by oterral** [comment]
@pfanguin the image is too compare to the old one big can you otpimize it pls

**Comment by pfanguin** [comment]
It's done

**Comment by pfanguin** [comment]
Test link : https://mf-geoadmin3.dev.bgdi.ch/fap_update_sprite/

**Comment by cedricmoullet** [comment]
nice ;-)

**Comment by cedricmoullet** [comment]
@davidoesch could you please review ?

File: src/components/topic/style/topic.less [hunk 1]
```diff
@@ -80,14 +80,50 @@
...
 
+  .ga-topics-sprite-dev {
+    width: 110px;
+    height: 60px;
+    background-position: 0 -300px;
+    -webkit-animation: play 2.5s steps(28) infinite;
+       -moz-animation: play 2.5s steps(28) infinite; 
+        -ms-animation: play 2.5s steps(28) infinite;
+         -o-animation: play 2.5s steps(28) infinite;
+            animation: play 2.5s steps(28) infinite;
```
Comment (oterral) [inline src/components/topic/style/topic.less]: USe bootsrap mixin `.animation(play 2.5s steps(28) infinite)`

File: src/components/topic/style/topic.less [hunk 2]
```diff
@@ -80,14 +80,50 @@
...
+  @-o-keyframes play {
+    from { background-position:    0 -300px; }
+      to { background-position: -440px -300px; }
+  }
+
+  @keyframes play {
+    from { background-position:    0 -300px; }
+      to { background-position: -3080px -300px; }
+  }
+
```
Comment (oterral) [inline src/components/topic/style/topic.less]: Use also a mixin:
```
.play-frames () {
	from { background-position:    0 -300px; }
      to { background-position: -440px -300px; }
}
 @-webkit-keyframes play {.play-frames;}
 @keyframes play {.play-frames;}
```
No need of ms, o,and moz prefixed, see http://caniuse.com/#search=keyframes

**Comment by oterral** [comment]
Very nice. 
Only one thing concerning the size of the image. I'm sure you don't need to put all the images in the same line using percentage in the `keyframes`.

**Comment by davidoesch** [comment]
really nice @pfanguin ---  ok

**Comment by oterral** [comment]
jenkins test

**Comment by oterral** [comment]
@pfanguin can you squash the commits pls

File: src/components/topic/style/topic.less [hunk 3]
```diff
@@ -9,85 +9,109 @@
...
+  }*/
+
+  .ga-topics-sprite-emapis {
+    background-position: -220px -660px;
   }
 
   .ga-topics-sprite-dev {
-    background-position: -220px -240px;
+    width: 110px;
+    height: 60px;
```
Comment (oterral) [inline src/components/topic/style/topic.less]: width and height are different than the others?

**Comment by pfanguin** [comment]
It's done

**Comment by cedricmoullet** [comment]
For the record, the 3d illustration replaces the dev illustration. We'll have to adapt when the real 3D topic will be implemented.

**Pull request merged**